app/globals.css

```diff
@@ -16,7 +16,12 @@
   }
 }
 
+html {
+  font-size: 10px;
```

Comment:
Keep in mind that doing this affects all rem units in your project, 1 rem will be equivalent to 10px now. REM units take the font-size of the html 

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#:~:text=what%20you%20get.-,ems%20and%20rems,-em%20and%20rem